syntax = "proto3";

service Showtime {
  rpc GetMovies(Date) returns (ShowtimeData){}
  rpc GetTimes(Empty) returns (stream ShowtimeData){}
}
message Date{
  string date = 1;
}

message ShowtimeData{
  string date = 1;
  repeated string movies = 2;
}

message Empty{
}